### DISCUSSION TITLE
validation in usefomcontext

### DISCUSSION THREAD

**Discussion by NarenderSiri** [react-hook-form/react-hook-form#9349] [General] [answered]
multiple render in useformcontext.please help me

[codesandbox](https://codesandbox.io/s/reacthookformmultiplerendering-03hgyc?file

https://user-images.githubusercontent.com/48355455/200841234-96057144-dca9-40a7-84d6-e238ed0350e3.mp4

**Comment by NarenderSiri** [comment]
@Moshyfawn  please help me

**Comment by leapful** [comment, marked as answer]
Duplicated with https://github.com/react-hook-form/react-hook-form/discussions/9347

Please stop using cloned account to post issue or discussion. It won't make the issue to be solved faster, it only causes confusion for helpers.
